modules/base/admin/module.py

```diff
@@ -445,12 +447,18 @@ async def pumpkin_shutdown(self, ctx):
 
     @commands.guild_only()
     @commands.check(check.acl)
-    @commands.group(name="spamchannel", aliases=["spam"])
-    async def spamchannel(self, ctx):
+    @commands.group(name="botspam", aliases=["spam"])
```

Comment:
I'd remove this alias.

It's OK to create aliases for subcommands, but top-level command naming pool can be exhausted quite quickly, so I'm against creating multiple aliases for those. Especially when **spam** may be used for something else, as it is a common word.